internal/controller/reconciler/internal_service_auth.go

```
@@ -0,0 +1,37 @@
+package reconciler
+
+import (
+	v2 "github.com/wandb/operator/api/v2"
+	"github.com/wandb/operator/pkg/utils"
+)
+
+// serviceAccountIssuerUnknownReason marks a CR that can't be reconciled because
+// the cluster's service-account issuer is neither configured nor discoverable.
+const serviceAccountIssuerUnknownReason = "ServiceAccountIssuerUnknown"
```

golangci-lint (2.12.2)

[error] const serviceAccountIssuerUnknownReason is unused

(unused)